### PR TITLE
Bump vim to 8.0.1282

### PR DESCRIPTION
**Pull request by alarcher**
Updated patchset kindly provided by @kenmays ;)

### CODE REVIEW

File: components/editor/vim/Makefile [hunk 1]
```diff
@@ -79,6 +80,7 @@ $(X_VARIANT)/.configured:	CONFIGURE_OPTIONS += --with-vim-name=gvim
 $(X_VARIANT)/.configured:	CONFIGURE_OPTIONS += --with-x
 $(X_VARIANT)/.configured:	CONFIGURE_OPTIONS += --enable-gui=gtk2
 $(X_VARIANT)/.configured:	CONFIGURE_OPTIONS += --enable-gtk2-check
+$(X_VARIANT)/.configured:       CONFIGURE_OPTIONS += --with-terminal
```
Comment (pyhalov) [inline components/editor/vim/Makefile]: Is something wrong with spaces?

File: components/editor/vim/Makefile [hunk 2]
```diff
@@ -117,3 +119,4 @@ REQUIRED_PACKAGES += x11/library/libice
 REQUIRED_PACKAGES += x11/library/libsm
 REQUIRED_PACKAGES += x11/library/libx11
 REQUIRED_PACKAGES += x11/library/toolkit/libxt
+
```
Comment (pyhalov) [inline components/editor/vim/Makefile]: Empty line

File: components/editor/vim/Makefile (hunk 1 above)
Comment (pyhalov) [reply components/editor/vim/Makefile]: Should it be --enable-terminal ? Also, why this option is enabled only for X_VARIANT?

Comment (alarcher) [reply components/editor/vim/Makefile]: @kenmays

**Comment by alarcher** [comment]
@pyhalov Fixed spaces and option.

**Comment by pyhalov** [comment]
@alarcher Have you compared vim-core size and dependencies after this change?

**Pull request merged**